Source/WebCore/Modules/credentialmanagement/CredentialsContainer.cpp

```diff
@@ -109,7 +109,7 @@ bool CredentialsContainer::performCommonChecks(const Options& options, Credentia
     }
 
     if (options.signal && options.signal->aborted()) {
-        promise.reject(Exception { ExceptionCode::AbortError, "Aborted by AbortSignal."_s });
+        promise.rejectType<IDLAny>(options.signal->reason().getValue());
```

Comment:
Needs a test.

Reply:
We have tests, but as I noted in the comment — we cannot run these in WebKit.

Reply:
Unless, sorry. If there is another place I could be writing/adding tests. Please point me in a direction here. I will wait for CI, perhaps we have already.

Reply:
@cdumez I have updated this PR with tests. Turns out we did have tests, and I was just looking in the wrong place. Learning every day!